### DISCUSSION TITLE
OTLP exporter with apache skywalking

### DISCUSSION THREAD

**Discussion by muthuIntern** [open-telemetry/opentelemetry-specification#3647] [Q&A]
Hi Team,

Currently, we are using apache skywalking framework to perform tracing and things are working fine.
I have a new requirement that need to expose traces with OTLP format compatibility. 
Is it possible to integrate OTLP exporter into skywalking java collector to export traces compatible with OTLP standard ?

At least, It would be great, If you could help in writing custom OTLP exporter for standalone java application to expose traces in OTLP format

Thanks,
Muthu

**Comment by trask** [comment]
hi @muthuIntern!

> Is it possible to integrate OTLP exporter into skywalking java collector to export traces compatible with OTLP standard ?

I may not understand your question exactly, but guessing this may be a better question to ask in the skywalking community
